tests/conftest.py

```diff
@@ -4523,3 +4523,19 @@ def finalizer():
 
     request.addfinalizer(finalizer)
     return factory
+
+
+@pytest.fixture(scope="function", autouse=True)
```

Comment:
I don't think we need to define new fixures. 
For setting the defaults, we can do that in setting OCP MG collection to False by default should be done in ocs_ci.framework.pytest_customization.ocscilib.pytest_runtest_makereport

Reply:
correct. but I want to add option to change it for specific tests.. So I used `disable_ocs_mg_collection_on_failure`, `enable_ocp_mg_collection_on_failure` fixtures.
So I think for the next test we need to init them again.. I will run it locally

Reply:
I tested on my PC
We need to add this fixture:
```import logging
from ocs_ci.framework import config
class TestAB():
    def test_a(self, enable_ocp_mg_collection_on_failure):
        logging.info(f"OCS-MG={config.REPORTING.get('ocs_must_gather_collection')}")
        logging.info(f"OCP-MG={config.REPORTING.get('ocp_must_gather_collection')}")

    def test_b(self):
        logging.info(f"OCS-MG={config.REPORTING.get('ocs_must_gather_collection')}")
        logging.info(f"OCP-MG={config.REPORTING.get('ocp_must_gather_collectio```